src/request/export/ExportWords.js

```diff
@@ -78,26 +79,30 @@ class ExportWords extends Nimiq.Observable {
      */
     async _passphraseSubmitted(phrase) {
         document.body.classList.add('loading');
+        const passphraseBuffer = phrase ? Nimiq.BufferUtils.fromAscii(phrase) : undefined;
+        /** @type {Key?} */
+        let key = null;
         try {
-            const passphraseBuffer = phrase ? Nimiq.BufferUtils.fromAscii(phrase) : undefined;
-            const key = await KeyStore.instance.get(this._request.keyInfo.id, passphraseBuffer);
-            if (!key) {
-                this._reject(new Error('keyId not found'));
-            }
-
-            this.setKey(key);
-            this.fire(ExportWords.Events.EXPORT_WORDS_KEY_CHANGED, {
-                key,
-                isProtected: this._request.keyInfo.encrypted,
-            });
-            window.location.hash = ExportWords.Pages.EXPORT_WORDS_SHOW_WORDS;
-            document.body.classList.remove('loading');
+            key = await KeyStore.instance.get(this._request.keyInfo.id, passphraseBuffer);
         } catch (e) {
             if (e.message === 'Invalid key') {
                 document.body.classList.remove('loading');
                 this._privacyWarningPassphraseBox.onPassphraseIncorrect();
-            } else this._reject(e);
+                return;
+            }
+            this._reject(new Errors.Core(e.message));
         }
+        if (!key) {
+            this._reject(new Errors.KeyIdNotFound());
+        }
+
+        this.setKey(key);
+        this.fire(ExportWords.Events.EXPORT_WORDS_KEY_CHANGED, {
+            key,
+            isProtected: this._request.keyInfo.encrypted,
+        });
+        window.location.hash = ExportWords.Pages.EXPORT_WORDS_SHOW_WORDS;
```

Comment:
Both the event and the page constant can be shortened to `KEY_CHANGED` and `SHOW_WORDS` respectively.

Reply:
For the event constant it's ok, but for the page constants no other handler uses prefixed constants.